cmd/helm/repo_remove_test.go

```diff
@@ -62,15 +62,7 @@ func TestRepoRemove(t *testing.T) {
 		t.Error(err)
 	}
 
-	idx := filepath.Join(rootDir, helmpath.CacheIndexFile(testRepoName))
-	mf, _ := os.Create(idx)
-	mf.Close()
-
-	idx2 := filepath.Join(rootDir, helmpath.CacheChartsFile(testRepoName))
-	mf, _ = os.Create(idx2)
-	mf.Close()
-
```

Comment:
The above code is cryptic.  What it means to do is to get a reference to the indexFile and to the cacheFile, so that after the `repo remove` command , we have a handle to check that both files no longer exist.